pkg/license/identifier.go

```diff
@@ -102,3 +102,25 @@ func Identify(pkgPath, content string) (string, error) {
 		return "", fmt.Errorf("cannot identify license content")
 	}
 }
+
+// GetLicenseContent from license id
+func GetLicenseContent(spdxID string) (string, error) {
+	for _, dir := range templatesDirs {
```

Comment:
`templatesDirs` includes `lcs-templates` and `header-templates`, I think in this case we only need to walk through `lcs-templates`